variables.tf

```diff
@@ -18,6 +18,25 @@ variable "okta_domain" {
   default   = ""
   sensitive = true
 }
+variable "okta_prefix" {
+  type      = string
+  default   = ""
+  sensitive = true
+}
+
+variable "okta_private_key" {
+  type      = string
+  default   = ""
+  sensitive = true
+}
+
+variable "okta_jwk_n" {
```

Comment:
did it this way because it was easier to get working, but should fully flesh this out as the full jwk, or even the full pub/priv key and read fields from it at a time.